using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -310,10 +334,236 @@ iOS simulators do not support push messaging.
 {% endhint %}
 
 To obtain the registration ID/token, see [Configuring Remote Notification Support](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1).
```

Comment:
Should you remove this as well since you are covering the details in set up description?